src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;
+
+    public Square(int firstSide, Color color) {
+        super(color);
+        this.firstSide = firstSide;
+    }
+
+    public void setFirstSide(int firstSide) {
+        this.firstSide = firstSide;

```

Comment:
The name of 'setFirstSide' method is confusing, it would be better to name it 'setSide' for the same reason as above.